.vscode/settings.json

```diff
@@ -66,6 +66,6 @@
   "restructuredtext.confPath": "",
   "python.linting.enabled": true,
   "python.linting.flake8Path": "${workspaceFolder}/.venv/bin/flake8",
-  "python.linting.pep8Path": "${workspaceFolder}/.venv/bin/pep8",
+  "python.linting.pycodestylePath": "${workspaceFolder}/.venv/bin/pep8",
```

Comment:
sigh didnt see this got stuffed in here - hopefully this isnt a version concern and the switch is correct